pkg/resources/manifests/deploy.yaml

```diff
@@ -4,6 +4,8 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: gadget
+  labels:
+    k8s-app: gadget
```

Comment:
Is this label also added when installing with helm?  (perhaps another option is to lookup for daemonset set instead)

Reply:
Thanks for bringing `helm` up. Unfortunately it seems like modifying or adding labels to the created namespace is not supported https://github.com/helm/helm/issues/3503

I will have to go another way. Probably iterating through every namespace and looking for a gadget daemonset

Reply:
I just pushed something so the helm chart seemlesly works too. I will look further into it tomorrow regarding the label why it was chosen to be `{{ include "gadget.fullname" . }}` before